crates/mapdiffbot2/config.example.toml

```diff
@@ -21,7 +21,7 @@ logging = "info"
 # Github credentials (Required)
 [github]
 app_id = 123456
-private_key_path = "diffbot.pem"
+private_key_path = "./config/diffbot.pem"
```

Comment:
Is this absolute or relative now? Maybe a comment on that

Reply:
it has always been relative to the cwd

Reply:
> it has always been relative to the cwd

Figured.

I doubt anyone but me is gonna host these for like the next 2 years minimum, but I might add a note when I do handover.